pkg/api/mlflow/config/config_test.go

```diff
@@ -17,19 +17,19 @@ func TestServiceConfig_Validate_Ok(t *testing.T) {
 		{
 			name: "ArtifactRootHasS3Prefix",
 			providedConfig: &ServiceConfig{
-				ArtifactRoot: "s3://bucket_name",
+				DefaultArtifactRoot: "s3://bucket_name",
```

Comment:
Please update all the test case names as well

Reply:
ahh, shit. yes.